From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mayagr <mayagr@qti.qualcomm.com>
Date: Tue, 28 Jul 2026 16:22:34 +0530
Subject: [PATCH 102/109] Cygwin: exit_process: narrow the stale TODO to the
 missing ARMNT case

getprocaddrarm64.exe is now provided, so the "provide exes for these"
TODO only applies to the 32-bit ARM (ARMNT) helper.  Reword the comment
accordingly.

(cherry picked from commit 345e3682340d0245b372b4bb52672fac5b4ac3a2)
(cherry picked from commit ddddb52c29c29c0bb705580ef3cbae8d190c5fa1)
(cherry picked from commit 8cb520b36394f5d0c49f25dd6f3e5d72eabc8d1d)
---
 winsup/cygwin/include/cygwin/exit_process.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/winsup/cygwin/include/cygwin/exit_process.h b/winsup/cygwin/include/cygwin/exit_process.h
index 0486a0c74..1a70bd471 100644
--- a/winsup/cygwin/include/cygwin/exit_process.h
+++ b/winsup/cygwin/include/cygwin/exit_process.h
@@ -69,7 +69,7 @@ kill_via_console_helper (HANDLE process, wchar_t *function_name, int exit_code,
     case IMAGE_FILE_MACHINE_AMD64:
       name = "/usr/libexec/getprocaddr64.exe";
       break;
-    /* TODO: provide exes for these */
+    /* TODO: provide a getprocaddrarm32.exe for ARMNT targets */
     case IMAGE_FILE_MACHINE_ARMNT:
       name = "/usr/libexec/getprocaddrarm32.exe";
       break;
